arwen/contexts/output_test.go

```diff
@@ -460,6 +460,45 @@ func TestOutputContext_Transfer_Errors_And_Checks(t *testing.T) {
 	require.Equal(t, big.NewInt(1000), blockchainContext.GetBalanceBigInt(sender))
 }
 
+func TestOutputContext_Transfer_IsAccountPayable(t *testing.T) {
+	t.Parallel()
+
+	sender := []byte("sender")
+	receiverNonPayable := make([]byte, 32)
```

Comment:
Also add a new SC address to the test, `receiverPayable`, and add another call to `outputContext.Transfer()` from `sender` to `receiverPayable` which should work.

Reply:
done